app/src/main/java/org/stepik/android/presentation/course_content/CourseContentPresenter.kt

```diff
@@ -408,6 +413,40 @@ constructor(
             )
     }
 
+    fun getCalendarPrimaryItems() {
```

Comment:
It's better to call it in different way as get method should have some return value. Lets rename it to `fetch...` like other similar methods.